infra/monitoring.tf

```diff
@@ -0,0 +1,10 @@
+module "monitoring" {
+
+  source = "./monitoring"
+
+  aws_region_name = var.aws_region
+  aws_account_id = var.aws_account_id
+  image_tag = "0.0.1"
+  database_name = "hudi"
+  table_name = "ticker_hudi_mor_ro"
+}
```

Comment:
Maybe add this to main.tf rather than a single file

Reply:
I prefer having it separately, right now it is now one resource for @alfonjerezi to have an example but there will be as many as tables you want to query for metrics.

Reply:
Ok that sounds good then